sdks/python/apache_beam/runners/interactive/extensions/apache-beam-jupyterlab-sidepanel/src/SidePanel.ts

```diff
@@ -119,6 +130,7 @@ export class SidePanel extends BoxPanel {
     this.dispose();
   }
 
+  private _clusters: ClustersWidget;
   private _inspector: InteractiveInspectorWidget;
```

Comment:
Once you pass in the widget, you can also turn this into a single variable.

Reply:
I've combined the two into a single variable now. Thanks!